source/js/buyers-guide/components/criterion/criterion.jsx

```diff
@@ -29,7 +29,7 @@ export default class Criterion extends React.Component {
               <button onClick={() => this.toggle()} className={this.state.helptextVisible ? `open` : `closed`}></button>
             }
           </p>
-          <p className="rating">{ meta.answer } <span className="emoji"></span></p>
+          <p className="rating"><span dangerouslySetInnerHTML={{__html: meta.answer}}></span> <span className="emoji"></span></p>
```

Comment:
Filed https://github.com/mozilla/foundation.mozilla.org/issues/2006 to look at this later.